src/App.js

```diff
@@ -1,27 +1,15 @@
-import logo from './logo.svg';
-import './App.css';
+/* eslint-disable react/prefer-stateless-function */
```

Comment:
- [ ] Please make sure you are not disabling eslint rules. These are here to help you write clean code.